src/pages/Content/ControlBar.tsx

```diff
@@ -59,19 +59,22 @@ function RenderActionText({ action }: { action: Action }) {
     <>
       {action?.type === 'click'
         ? `Click on ${action.tagName.toLowerCase()} ${getBestSelectorForAction(
-            action
+            action,
+            ScriptType.Playwright
```

Comment:
Should we pass scriptType to this method instead of using a fixed value ?

Reply:
The user's scriptType really isn't set if the ControlBar is collapsed (to the user anyways), I think it'd only be a marginal gain. Long term this UX does need to be figured out but for now I think it's okay hardcoding to be Playwright.

Reply:
Sounds good